src/Engine/Renderer/Camera/Camera.cpp

```diff
@@ -17,7 +17,10 @@ using Core::Math::PiDiv4;
 namespace Engine {
 
 Camera::Camera( Entity* entity, const std::string& name, Scalar height, Scalar width ) :
-    Component( name, entity ), m_width{width}, m_height{height}, m_aspect{width / height} {}
+    Component( name, entity ),
+    m_width{width},
+    m_height{height},
+    m_aspect{width / height} {}
```

Comment:
It seems that this modification do not correspond to the provided clang-format.
Please update your editor configuration (see in `script` directory, `clang-format` and `install-scripts-linux.sh`)

Reply:
It seems that I have the same effect using the provided clang format.